catalog/spatial/geowebcache/README.md

```diff
@@ -59,7 +59,7 @@ To update a configuration with CURL :
 GeoWebCache also provides a user interface that can be used to preview layers and truncate or seed them at `https://localhost:8993/geowebcache/`.
 
 ### Configure GWC disk quota
-> Since disk usage increases geometrically by zoom level, one single seeding task could fill up an entire storage device. Because of this, GeoWebCache employs a disk quota system where one can specify the maximum amount of disk space to use for a particular layer or for the entire set of layers (the “Global Quota”), as well as logic on how to proceed when that quota is reached. There are two different policies for managing the disk quotas: Least Frequently Used (LFU) and Least Recently Used (LRU). 
+> Since disk usage increases geometrically by zoom level, one single seeding task could fill up an entire storage device. Because of this, GeoWebCache employs a disk quota system where one can specify the maximum amount of disk space to use for a particular layer or for the entire set of layers (the “Global Quota”), as well as logic on how to proceed when that quota is reached. There are two different policies for managing the disk quotas: Least Frequently Used (LFU) and Least Recently Used (LRU).
```

Comment:
Is the only change here a space at the end of the line? If so can we revert this change since it isn't related to the rest of the change set.

Reply:
Yeah, this was the formatter.